benchmark/benchmark_scenarios.py

```diff
@@ -0,0 +1,55 @@
+from twisted.internet.defer import maybeDeferred, Deferred, succeed
+
+
+class RunningScenario:
+
+    def __init__(self):
+        self.scenario_established = Deferred()
+        self.scenario_maintained = Deferred()
```

Comment:
Why prefix these with `scenario`? Shouldn't they be `_established` and `_maintained`? :grey_question: